src/tests/ftest/dfuse/daos_build.py

```diff
@@ -138,8 +138,8 @@ def run_build_test(self, cache_mode, intercept=False, run_on_vms=False):
         remote_env = {}
         if run_on_vms:
             dfuse_namespace = dfuse_namespace = "/run/dfuse_vm/*"
-            build_jobs = 6 * 2
-            remote_env['D_IL_MAX_EQ'] = '6'
+            build_jobs = 6
+            remote_env['D_IL_MAX_EQ'] = '2'
```

Comment:
This will reduce the maximum number of concurrent contexts from 72 to 12 which is probably heavy-handed for fixing the issue but with seemingly different limits on different clusters and the need to fix this test I'm unable to suggest either a better number or a reasonable process for discovering a higher number that would allow more throughput without overwhelming the system.